packages/packages-groonga-org-package-task.rb

```diff
@@ -243,6 +246,12 @@ def define_release_tasks
             download_packages(target_namespace) if enabled
           end
           tasks << "#{target_namespace}:download"
+
+          desc "Make .htaccess for #{target_namespace} packages"
+          task :create_htaccess do
+            create_htaccess(target_namespace) if enabled
+          end
+          tasks << "#{target_namespace}:create_htaccess"
```

Comment:
It seem that this isn't related to `use_built_package?`.

Reply:
I will fixed it.

I focused on the case of redirecting URL so it added to this line because there should be some built packages to redirect.
But it seems that `create_htaccess` ( will be renamed `prepare`) is for not only redirection but also some other cases.
So it should be out of `use_built_package?`

Reply:
I have fixed it.